### PR TITLE
implement task 1

### PR DESCRIPTION
**Pull request by jjanickamatraszek**
add basic pages for HomePage, MainMenu, NewInSubMenu
implement one sample test scenario (display submenu of 'New In' menu category)

### CODE REVIEW

**Review by Trukhanovich — APPROVED** [review]
lgtm

File: src/main/java/com/solvd/components/MainMenu.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class MainMenu {
+    private final WebDriver driver;
+
+    @FindBy(xpath = ".//ul[@data-testid='category-list']//span[contains(text(),'new in')]/parent::li")
```
Comment (kzukowski-solvd) [inline src/main/java/com/solvd/components/MainMenu.java:10]: According to the 5th point in the task specification web elements should have css selectors.

File: src/main/java/com/solvd/components/NewInSubMenu.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.List;
+
+public class NewInSubMenu {
+    private final WebDriver driver;
+
+    @FindBy(xpath = ".//span[contains(text(),'new in')]/parent::li//ul[contains(@class,'inner-list')]")
```
Comment (kzukowski-solvd) [inline src/main/java/com/solvd/components/NewInSubMenu.java:13]: xpath -> css

File: src/main/java/com/solvd/components/NewInSubMenu.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    private WebElement submenuContainer;
+
+    @FindBy(xpath = ".//span[contains(text(),'new in')]/parent::li//ul/div[contains(@class,'photo-section')]")
```
Comment (kzukowski-solvd) [inline src/main/java/com/solvd/components/NewInSubMenu.java:16]: xpath -> css

**Review by kzukowski-solvd — CHANGES_REQUESTED** [review]
Some changes ;P

File: src/main/java/com/solvd/components/MainMenu.java (hunk 1 above)
Comment (jjanickamatraszek) [reply src/main/java/com/solvd/components/MainMenu.java:10]: I use text() method from xpath to be sure that 'new in' element is clicked and text() is not supported by css

File: src/main/java/com/solvd/components/NewInSubMenu.java (hunk 2 above)
Comment (jjanickamatraszek) [reply src/main/java/com/solvd/components/NewInSubMenu.java:13]: the same case as above -> i use text() from xpath

File: src/main/java/com/solvd/components/NewInSubMenu.java (hunk 3 above)
Comment (jjanickamatraszek) [reply src/main/java/com/solvd/components/NewInSubMenu.java:16]: the same case as above -> i use text() from xpath

**Pull request merged**